mapbox/libjava-core/src/main/java/com/mapbox/services/commons/models/Position.java

```diff
@@ -27,24 +28,27 @@
    * @since 1.0.0
    */
   private Position(double longitude, double latitude, double altitude) {
+    // Used to remove any trailing zeros and prevent a coordinate being over 7 significant figures.
+    DecimalFormat decimalFormat = new DecimalFormat("0.######");
```

Comment:
Because this same logic is used in several place, how about exposing it as static utility method?